geth/common/utils.go

```diff
@@ -143,6 +144,11 @@ func Fatalf(reason interface{}, args ...interface{}) {
 
 // CreateTransaction returns a transaction object.
 func CreateTransaction(ctx context.Context, args SendTxArgs) *QueuedTx {
+	if args.Input == nil && args.Data == nil {
```

Comment:
I don't get why we need this condition.

Reply:
This was more of a sanity check during the migration that slipped through.